src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -541,6 +532,37 @@ private static void saveColonies()
         saveNeeded = false;
     }
 
+    public static boolean backupColonyData()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)